firebase-appdistribution/test-app/test-app.gradle

```diff
@@ -81,6 +93,7 @@ dependencies {
 
     // Beta flavor uses the full implementation
     betaImplementation project(':firebase-appdistribution')
+//    betaImplementation 'com.google.firebase:firebase-appdistribution:16.0.0-beta08'
```

Comment:
delete outright instead of adding comment?

Reply:
Sorry yes please refresh